docs/source/upgrade-guide/index.md

```diff
@@ -89,6 +89,26 @@ If you want to retain the old behavior (and no use `apiExpanders` at all), you n
 config.settings.apiExpanders = [];
 ```
 
+### Cypress upgraded to 12.17.1
+
+As usual in a Volto major version, Cypress has been upgraded to latest version to date.
+There are no major changes to the way the tests are implemented and run.
+
+However, could be that the Cypress boilerplate you are using has to be updated in your projects and add-ons in case you are using `@testing-library/cypress` in your tests.
+This is due to a change in how the default commands are now built internally and `@testing-library/cypress`.
+
+You need to move the import:
+
+```js
+import '@testing-library/cypress/add-commands';
+```
+
+from `cypress/support/commands.js` to `cypress/support/e2e.js`, in case you have it in there.
+
+This is because the overrides that the `@testing-library/cypress` introduces can only be run once.
+Since there are some commands that can call exports in `cypress/support/commands.js`, this import was run more than once, then it errors.
```

Comment:
It would be good to provide an example of the explicit error message for SEO. "Google for the error."

```suggestion
Since there are some commands that can call exports in {file}`cypress/support/commands.js`, this import may be run more than once, and then it errors.
```